packages/ckeditor5-engine/src/view/renderer.ts

```diff
@@ -1163,3 +1227,35 @@ function createFakeSelectionContainer( domDocument: DomDocument ): DomElement {
 
 	return container;
 }
+
+function updateTextNode( domText: DomText, expectedText: string ) {
```

Comment:
Docs needed: what's the general algorithm it follows.

Reply:
**And why it does what it does.**

Reply:
I can see it's a copy paste from another place. So perhaps there's not much "why". But I'd still prefer if we had at least one sentence of what it does.